src/Components/App/App.test.js

```diff
@@ -639,23 +625,23 @@ describe('App', () => {
       content_selected: "MOVEMENT",
       focus_interval: "30",
       rest_interval: "5",
+      user_id: 3
     });
 
     const { getByRole } = render(
       <MemoryRouter>
-        <UserContext.Provider
-          value={[{ userName: "Fran", token: "200", userId: 3 }]}
-        >
-          <SettingsProvider>
+        <SettingsContext.Provider value={[{ workInterval: "45", breakInterval: "5", sound: "reverbSplash", moodRating: true }, jest.fn()]}>
+          <UserContext.Provider value={[{ userName: "Fran", email: "fran@gmail.com", userId: 3 }]}
+          >
```

Comment:
Way to use this technique to trick the tests. Thank you for putting in so much time and energy to make this work!